django/db/backends/base/features.py

```diff
@@ -24,6 +24,7 @@ class BaseDatabaseFeatures:
     supports_partially_nullable_unique_constraints = True
     # Does the backend support initially deferrable unique constraints?
     supports_deferrable_unique_constraints = False
+    supports_db_comments = False
```

Comment:
I think the `_db` is redundant here - it's quite clear it refers to the database since we're in database features.